EnglishTranslation-master/ITEM.tsv

```diff
@@ -10195,44 +10195,44 @@ ITEM_20160224_010194	Rune Stone of Destruction
 ITEM_20160224_010195	Rune Stone of the First Master
 ITEM_20160224_010196	A rune stone made by the first master of the Rune Casters.
 ITEM_20160224_010197	Blue Magic Stone
-ITEM_20160224_010198	Material of a rune stone. In order to become a Rune Caster, you must make a rune stone of your own.
+ITEM_20160224_010198	Material of a rune stone. In order to become a Rune Caster, you must create a rune stone of your own.
 ITEM_20160224_010199	Malstatue's Magic Fragment
 ITEM_20160224_010200	Velaphid's Cold Blood
 ITEM_20160224_010201	Pumpleflap's Robust Horn
-ITEM_20160310_010202	Lolopanthers are beings who at a particular time period have served Goddess Zemyna. Although most of them are human, not all of them are.
-ITEM_20160310_010203	Not many records remain of those who have worshiped Goddess Zemyna throughout the eras. A few older locations retain traces of this goddess' history. The names given to these locations are an example of this. The same can be said particularly for weapons.
-ITEM_20160310_010204	As the only way left to meet Goddess Zemyna was to climb up the Earth Tower, this allowed more ways to reconstruct history, as well as better opportunities to obtain magical weapons.
-ITEM_20160310_010205	The cost to maintain this bow is pricey, except it is useful for those who serve the Fletcher Master for free. People like Lydia Schaffen, the one who founded the Astral Tower.
+ITEM_20160310_010202	Lolopanthers are beings who once served Goddess Zemyna. Although most of them are human, not all of them are.
+ITEM_20160310_010203	Not many records remain of those who have worshiped Goddess Zemyna throughout time. A few older locations retain traces of this goddess' past. The names given to these locations are an example of this. The same can be said particularly for weapons embued with her touch.
```

Comment:
Change it to `Not many records remain of those who served Goddess Zemyna in their times. But we can witness sections of the goddess' history through names such as those of a few historical places; and especially the names of weapons such as this.`

Thanks to Kei.